flang/test/Transforms/debug-complex-1.fir

```diff
@@ -0,0 +1,39 @@
+// RUN: fir-opt --cg-rewrite="preserve-declare=true" --mlir-print-debuginfo %s | fir-opt --add-debug-info --mlir-print-debuginfo | FileCheck %s
```

Comment:
Does the cg-rewrite pass do anything useful here?

Reply:
No, only add-debug-info is needed. I have updated the test.